src/tools/spin/spinquic.cpp

```diff
@@ -1125,58 +1336,9 @@ main(int argc, char **argv)
     }
     printf("Using seed value: %u\n", RngSeed);
     srand(RngSeed);
+    start();
 
-    //
-    // Initial MsQuicOpen2 and initialization.
-    //
-    const QUIC_API_TABLE* TempMsQuic = nullptr;
-    ASSERT_ON_FAILURE(MsQuicOpen2(&TempMsQuic));
-    CxPlatCopyMemory(&MsQuic, TempMsQuic, sizeof(MsQuic));
-
-    if (Settings.AllocFailDenominator > 0) {
-        if (QUIC_FAILED(
-            MsQuic.SetParam(
-                nullptr,
-                QUIC_PARAM_GLOBAL_ALLOC_FAIL_DENOMINATOR,
-                sizeof(Settings.AllocFailDenominator),
-                &Settings.AllocFailDenominator))) {
-            printf("Setting Allocation Failure Denominator failed.\n");
-        }
-    }
-
-    if (Settings.LossPercent != 0) {
-        QUIC_TEST_DATAPATH_HOOKS* Value = &DataPathHooks;
-        if (QUIC_FAILED(
-            MsQuic.SetParam(
-                nullptr,
-                QUIC_PARAM_GLOBAL_TEST_DATAPATH_HOOKS,
-                sizeof(Value),
-                &Value))) {
-            printf("Setting Datapath hooks failed.\n");
-        }
-    }
-
-    MsQuicClose(TempMsQuic);
-
-    Settings.RunTimeMs = Settings.RunTimeMs / RepeatCount;
-    for (uint32_t i = 0; i < RepeatCount; i++) {
-
-        CXPLAT_THREAD_CONFIG Config = {
-            0, 0, "spin_run", RunThread, nullptr
-        };
-        CXPLAT_THREAD Threads[4];
-        const uint32_t Count = (uint32_t)(rand() % (ARRAYSIZE(Threads) - 1) + 1);
-
-        for (uint32_t j = 0; j < Count; ++j) {
-            ASSERT_ON_FAILURE(CxPlatThreadCreate(&Config, &Threads[j]));
-        }
-
-        for (uint32_t j = 0; j < Count; ++j) {
-            CxPlatThreadWait(&Threads[j]);
-            CxPlatThreadDelete(&Threads[j]);
-        }
-    }
-
-    CxPlatLockUninitialize(&RunThreadLock);
     return 0;
 }
+
+#endif
```

Comment:
Can you put a comment indicated what #if this is ending?

Reply:
Done